crypto/getenv.c

```diff
@@ -91,7 +135,18 @@ char *ossl_safe_getenv(const char *name)
 #if defined(__GLIBC__) && defined(__GLIBC_PREREQ)
 # if __GLIBC_PREREQ(2, 17)
 #  define SECURE_GETENV
-    return secure_getenv(name);
+#   /* 
```

Comment:
is that # supposed to be there?  I've not tried but it seems like that won't compile

Reply:
No. However it compiles for me, and with those definitions active (thus using my new code). Odd. However I've removed and will retest. thanks!

Reply:
fixed